app/code/Inchoo/Sample08/Plugin/CollectionPlugin.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inchoo\Sample08\Plugin;
+
+use Magento\Catalog\Model\ResourceModel\Product\Collection;
+
+/**
+ * Sets the $printQuery parameter to false before calling the load() method in Collection
+ */
+class CollectionPlugin
+{
+    /**
+     * @param Collection $subject
+     * @param bool $printQuery
+     * @param bool $logQuery
+     * @return array
+     */
+    public function beforeLoad(Collection $subject, bool $printQuery, bool $logQuery): array
+    {
+        return [false, $logQuery];
```

Comment:
radi i ovo, ali u zadataku piše "set the `printQuery` argument to `false` if it is not already"